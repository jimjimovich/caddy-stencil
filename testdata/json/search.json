[{"title":"San Francisco","location_type":"City","woeid":2487956,"latt_long":"37.777119, -122.41964"},{"title":"San Diego","location_type":"City","woeid":2487889,"latt_long":"32.715691,-117.161720"}]