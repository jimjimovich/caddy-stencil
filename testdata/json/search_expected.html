<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Search For A City</title>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.7.1/css/bulma.min.css" type="text/css" />
  </head>
  <body>
    <section class="section">
      <div class="container">
        
          <h1 class="title is-1">Search Results</h1>
          <ul style="margin-bottom: 20px;">
            
              <li><a href="/api/location/2487956/">San Francisco</a></li>
            
              <li><a href="/api/location/2487889/">San Diego</a></li>
            
          </ul>
        
        <div class="columns">
          <div class="column">
            <form action="/api/location/search/">
              <div class="field has-addons">
                <div class="control">
                  <input class="input" type="text" name="query" placeholder="Find a city">
                </div>
                <div class="control">
                  <input class="button is-info" type="submit" value="Search"/>
                </div>
              </div>
            </form>
          </div>
          <div class="column">
            <p class="has-text-right">data provided by <a href="https://www.metaweather.com/">metaweather.com</a></p>
          </div>
        </div>
      </div>
    </section>
    .Doc.body should be empty
  </body>
</html>